spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -160,6 +160,12 @@
       then_I_should_see "Scheduled to be published at"
       and_there_should_be_no_schedule_button
     end
+
+    scenario "User unschedules publishing" do
+      given_there_is_a_scheduled_step_by_step_page
+      and_I_visit_the_publish_or_delete_page
+      then_I_see_an_unschedule_button
```

Comment:
Suggest renaming to `then_I_should_see_an_unschedule_button` or `then_there_should_be_an_unschedule_button`.

Reply:
I went for this naming convention because there are 8 other methods in this test file with names starting with `then_I_see`.

Reply:
Fair enough. I might go in later and raise a renaming PR to have the English flow better :)